switch.py

```
@@ -0,0 +1,325 @@
+        odin_secret_access_key = input("Enter ODIN_SECRET_ACCESS_KEY: ").strip()
+        # Call configure for both old and new odin
+        configure()
+        exit(0)
+
+
+def get_current_bin_version():
+    try:
+        files = os.listdir(INSTALL_DIR)
+        for file in files:
+            if file.startswith('odin-'):
+                return file.split("-")[1]
+    except FileNotFoundError:
+        return None
+
+
+def update_binary():
+    url = f"https://api.github.com/repos/dream11/odin/git/trees/{branch}?recursive=1"
+
+    req = Request(url)
+    req.add_header('Accept', 'application/json')
+    try:
+        response = urlopen(req).read()
+        if response:
+            data = json.loads(response)
+            files = [item["path"] for item in data.get("tree", []) if item["type"] == "blob"]
+
+            # Filter files that start with "odin"
+            filtered_files = [file for file in files if file.startswith('odin-')]
+            latest_version = filtered_files[0].split("-")[1]
+
+            current_version = get_current_bin_version()
+
+            if current_version is None or current_version < latest_version:
+                print(f"Updating odin binary to version {latest_version}")
+                url = f"https://raw.githubusercontent.com/dream11/odin/{branch}/odin-{latest_version}"
+                filepath = os.path.join(INSTALL_DIR, f"odin-{latest_version}")
+                urlretrieve(url, filename=filepath)
+                os.chmod(filepath, 0o755)
+                # Enable app verification and remove quarantine attributes
+                subprocess.call(["sudo", "spctl", "--master-enable"])
+                subprocess.call(["xattr", "-dr", "com.apple.quarantine", filepath])
+        else:
+            print(f"Error: Unable to fetch files (Status Code: {response.status_code})")
+            return []
+    except Exception as e:
+        print(f"Error: Unable to fetch files: {e}")
```

Why swallowing exception?